src/main/java/hudson/plugins/cobertura/MavenCoberturaPublisher.java

```diff
@@ -210,7 +170,7 @@ public boolean postExecute(final MavenBuildProxy build, final MavenProject pom,
     }
 
     private boolean isCoberturaReport(MojoInfo mojo) {
-        if (!mojo.pluginName.matches("org.codehaus.mojo", "cobertura-maven-plugin") || !mojo.pluginName.matches("org.codehaus.mojo", "cobertura-it-maven-plugin"))
+        if (!mojo.pluginName.matches("org.codehaus.mojo", "cobertura-maven-plugin") && !mojo.pluginName.matches("org.codehaus.mojo", "cobertura-it-maven-plugin"))
```

Comment:
As it turns out, this entire integration has been broken for years, since #8, so we could also delete `MavenCoberturaPublisher` and the `maven-plugin` dependency without loss of functionality relative to current releases.

Reply:
Deleting it makes sense to me. Do you want to do the honors, or should I?

Reply:
Oh no, let me do it, it would be **my pleasure**. PR in a few minutes if all goes well…

Reply:
Sorry - deleted my original comment, since I accidentally used the wrong github identity. To avoid future confusion, I said:

`Deleting it makes sense to me. Do you want to do the honors, or should I?`